gcloud/monitoring/label.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Label Descriptors for the Google Monitoring API."""
+
+import collections
+
+
+class LabelDescriptor(collections.namedtuple('LabelDescriptor',
+                                             'key value_type description')):
+    """Schema specification and documentation for a single label.
+
+    :type key: string
+    :param key: The name of the label.
+
+    :type value_type: string
+    :param value_type: The type of the label. It must be one of ``"STRING"``,
```

Comment:
About 90% of the string parameters in the existing code base are described as `string` rather than `str`. It seems more readable. More importantly, `str` is overly specific. If this were a type declaration, it would be `six.string_types`.

I'm new to sphinx, so I'm trying to follow existing practice in `gcloud-python`. I didn't find a lot of other projects that declare types in docstrings using `:type`.